src/Compiler/Service/ServiceDeclarationLists.fs

```diff
@@ -224,14 +223,14 @@ module DeclarationListHelpers =
 
         | Item.RecdField rfinfo when rfinfo.TyconRef.IsFSharpException ->
             let ty, _ = PrettyTypes.PrettifyType g rfinfo.FieldType
-            let id = rfinfo.RecdField.Id
+            let id = rfinfo.DisplayName
```

Comment:
This should fix quick info when hovering over `thing with space` in 

```fsharp
exception SomeError1 of message: string
exception SomeError2 of ``thing with space``: string
```

Reply:
Yep, it does, here's the [issue](https://github.com/dotnet/fsharp/issues/13593), here's the [test](https://github.com/dotnet/fsharp/pull/13552/files#diff-1e6dc3c6f1f00f584bbc25dce0ef0259e996c14c91d1b58b58ace6e297325cabR495)